src/tactic/core.lean

```diff
@@ -353,6 +344,62 @@ do to_remove ← hs.mfilter $ λ h, do {
   to_remove.mmap' (λ h, try (clear h)),
   return (¬ to_remove.empty ∨ goal_simplified)
 
+/-- `revert_after e` reverts all local constants after local constant `e`. -/
+meta def revert_after (e : expr) : tactic ℕ := do
+  l ← local_context,
```

Comment:
```suggestion
l ← local_context,
```
This might just be a personal stylistic preference though.

Reply:
In most places, we do start indenting a `do` block, so I'll keep it like this. I personally don't really care.